src/lib/tooltip/tooltip.ts

```diff
@@ -212,20 +223,15 @@ export class MdTooltip implements OnDestroy {
   show(delay: number = this.showDelay): void {
     if (this.disabled || !this._message || !this._message.trim()) { return; }
 
-    if (!this._tooltipInstance) {
-      this._createTooltip();
-    }
-
-    this._setTooltipClass(this._tooltipClass);
-    this._setTooltipMessage(this._message);
+    this._scrollStrategy!.enable();
```

Comment:
It shouldn't be necessary to enable the scroll strategy manually, it gets enabled/disabled when the overlay is attached and detached.